Engine/AppManager.h

```diff
@@ -44,11 +44,9 @@ CLANG_DIAG_ON(deprecated)
 #include <QtCore/QProcess>
 #include <QtCore/QMap>
 
-#if !defined(Q_MOC_RUN) && !defined(SBK_RUN)
```

Comment:
why remove these? they are needed for shiboken1, and they don't hurt shiboken2 I think

Reply:
This hurts to the death the shiboken2 generation, but we can go by another way by doing something in the likes of `#define SBK_1_RUN` or see if the shiboken invocation emits defines during gen.

Reply:
yes, I'd prefer that